packages/edit-site/src/utils/is-previewing-theme.js

```diff
@@ -4,9 +4,7 @@
 import { getQueryArg } from '@wordpress/url';
 
 export function isPreviewingTheme() {
-	return (
-		getQueryArg( window.location.href, 'wp_theme_preview' ) !== undefined
-	);
+	return !! getQueryArg( window.location.href, 'wp_theme_preview' );
```

Comment:
@youknowriad Do you remember why you had to modify this? It looks like the `wp_theme_preview` query arg could be an empty string or some other falsy value, but the details are not clear. The old code, with its `!== undefined` check, returns `true` if the `wp_theme_preview` query arg is present, with any value.

Reply:
I remember it partially. I think it has to do with the "active theme" button. Basically when activating a theme, we actually set the `wp_theme_preview` to empty string (but keep in the url), for the site editor to adapt properly.

There's a reason it's kept in the url instead of undefined though but I don't remember the reason.